conf/plants/mlss_conf.xml

```diff
@@ -4,13 +4,37 @@
   <!-- Collections are species-sets that are needed to define several mlsss -->
   <collections>
 
+    <!-- All plants except the triticum_aestivum cultivars -->
+    <collection name="default">
+      <taxonomic_group taxon_name="Rhodophyta"/>
+      <taxonomic_group taxon_name="Viridiplantae">
+        <!-- But exclude all triticum_aestivum cultivars -->
+        <ref_for_taxon name="triticum_aestivum"/>
+      </taxonomic_group>
+      <genome name="caenorhabditis_elegans"/>
+      <genome name="ciona_savignyi"/>
+      <genome name="drosophila_melanogaster"/>
+      <genome name="homo_sapiens"/>
+      <genome name="saccharomyces_cerevisiae"/>
+    </collection>
+
```

Comment:
A couple of comments on this - for the `collection-default` - we want everything except the triticum_aestivum cultivars. So a simpler way to define might be:
```suggestion
    <!-- All plants except the triticum_aestivum cultivars -->
    <collection name="default">
      <taxonomic_group taxon_name="Eukaryota"/>
        <!-- But exclude all triticum_aestivum cultivars -->
        <ref_for_taxon name="triticum_aestivum"/>
    </collection>

```

Reply:
I have thought about that, but I was concerned that some unwanted species would get under our radar this way? But it makes sense to me. This would also remove the requirement of those 4 species in the allowed list (they are hardcoded in the `PrepareMasterDB` pipeline anyway).

Reply:
Yes, supposedly the only way they could sneak in is if they were included in the `allowed_species.json` file :shrug: